rpc_state_reader/src/lib.rs

```diff
@@ -187,6 +212,96 @@ impl RpcState {
         serde_json::from_str(&response.into_string().unwrap()).unwrap()
     }
 
+    /// Requests the given transaction to the Feeder Gateway API.
+    pub fn get_transaction(&self, hash: &str) -> Transaction {
+        let params = ureq::json!({
+            "jsonrpc": "2.0",
+            "method": "starknet_getTransactionByHash",
+            "params": [format!("0x{}", hash)],
+            "id": 1
+        });
+        let response: serde_json::Value = self.rpc_call(&params).unwrap();
+
+        match response["result"]["type"].as_str().unwrap() {
```

Comment:
Are these `unwrap`s safe? If so, could you add a `SAFETY:` comment explaining why? Otherwise, could we return errors if the condition doesn't hold?

Reply:
Oh, sorry, I missed that it's inside #[cfg(test)]. Panics in tests are OK.

Reply:
Yes, all these functions are only used in tests